.cirrus.yml

```diff
@@ -15,6 +16,17 @@ test_task:
       env:
         # This should have the same coverage as the previous task, so skip it.
         SKIP_UPLOAD_COVERAGE: 1
+    - name: Test on Linux (Gccgo)
```

Comment:
Maybe name this "Test on Debian using gccgo". Too, add a comment explaining that all programs written in Go are built using gccgo on that platform. Thus the desire to add a CI test to verify that using gccgo to build elvish works.

Reply:
The title is fine; the fact that it uses Debian is an implementation detail; the intention is to test gccgo on Linux.

> Too, add a comment explaining that all programs written in Go are built using gccgo on that platform.

That's not true.